### PR TITLE
Patches

### PR DESCRIPTION
**Pull request by y1bs**
# Feature
Refactor Cart

# Changelog:

# Merge/Review Order

These other PRs should be merged or reviewed first

This PR blocks these other PRs

## Checklist:

- [ ] Merged latest develop
- [ ] PR title makes sense

### CODE REVIEW

**Review by rawrrencee — DISMISSED** [review]
lgtm!! xxn

File: src/services/cart/index.js [hunk 1]
```diff
@@ -61,12 +61,16 @@ export async function checkOut(courseIds, mentorshipListingIds) {
...
-      withCredentials: true,
-    })
+    .post(
+      `/paypal/order/capture?paymentId=${paymentId}&token=${token}&PayerID=${payerID}&cartId=${cartId}`,
+      {
+        withCredentials: true,
+      },
+    )
     .then(response => {
+      console.log(response)
```
Comment (natalieloke) [inline src/services/cart/index.js]: xtra console log - can remove in future PRs

**Review by natalieloke — DISMISSED** [review]
awesome thx @y1bs for the fast patch!!

File: src/services/cart/index.js (hunk 1 above)
Comment (y1bs) [reply src/services/cart/index.js]: Ahh okok ps, must have missed this, will remove after this

**Review by rawrrencee — APPROVED** [review]

**Pull request merged**